packages/docusaurus-theme-classic/src/theme/BlogListPage/index.js

```diff
@@ -7,15 +7,21 @@
 
 import React from 'react';
 
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import Layout from '@theme/Layout';
 import BlogPostItem from '@theme/BlogPostItem';
 import BlogListPaginator from '@theme/BlogListPaginator';
 
 function BlogListPage(props) {
   const {metadata, items} = props;
+  const {
+    siteConfig: {title: siteTitle},
+  } = useDocusaurusContext();
+  const isBlogOnlyMode = metadata.permalink === '/';
```

Comment:
I'm not sure about this. How does this work?

Reply:
What is wrong? If the blog `routeBasePath` is equal to `/` then blog-only mode is used!
![image](https://user-images.githubusercontent.com/4408379/74084578-82910480-4a81-11ea-83bd-902bb7a514cf.png)

Reply:
I guess referencing the `permalink` field wasn't intuitive to me